diff --git a/com.prolan.tycho.release1/META-INF/MANIFEST.MF b/com.prolan.tycho.release1/META-INF/MANIFEST.MF
index c91d0fe..ef6824e 100644
--- a/com.prolan.tycho.release1/META-INF/MANIFEST.MF
+++ b/com.prolan.tycho.release1/META-INF/MANIFEST.MF
@@ -2,6 +2,6 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: Release1
 Bundle-SymbolicName: com.prolan.tycho.release1
-Bundle-Version: 1.0.0.qualifier
+Bundle-Version: 1.1.1
 Export-Package: com.prolan.tycho.release1
-Require-Bundle: com.prolan.tycho.release2;bundle-version="1.0.0"
+Require-Bundle: com.prolan.tycho.release2;bundle-version="1.1.1"
diff --git a/com.prolan.tycho.release2/META-INF/MANIFEST.MF b/com.prolan.tycho.release2/META-INF/MANIFEST.MF
index 7831a87..d721414 100644
--- a/com.prolan.tycho.release2/META-INF/MANIFEST.MF
+++ b/com.prolan.tycho.release2/META-INF/MANIFEST.MF
@@ -2,6 +2,6 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: Release2
 Bundle-SymbolicName: com.prolan.tycho.release2
-Bundle-Version: 1.0.0.qualifier
+Bundle-Version: 1.1.1
 Export-Package: com.prolan.tycho.release2
-Require-Bundle: com.prolan.tycho.release3;bundle-version="1.0.0"
+Require-Bundle: com.prolan.tycho.release3;bundle-version="1.1.1"
diff --git a/com.prolan.tycho.release3/.project b/com.prolan.tycho.release3/.project
index 48df7c9..2190165 100644
--- a/com.prolan.tycho.release3/.project
+++ b/com.prolan.tycho.release3/.project
@@ -25,4 +25,11 @@
 		<nature>org.eclipse.jdt.core.javanature</nature>
 		<nature>org.eclipse.pde.PluginNature</nature>
 	</natures>
+	<linkedResources>
+		<link>
+			<name>pom.xml</name>
+			<type>1</type>
+			<location>C:/Users/fekete.agnes/my_github2/tycho_release_scm/pom.xml</location>
+		</link>
+	</linkedResources>
 </projectDescription>
diff --git a/com.prolan.tycho.release3/META-INF/MANIFEST.MF b/com.prolan.tycho.release3/META-INF/MANIFEST.MF
index 93b0c8b..0c75201 100644
--- a/com.prolan.tycho.release3/META-INF/MANIFEST.MF
+++ b/com.prolan.tycho.release3/META-INF/MANIFEST.MF
@@ -2,5 +2,5 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: Release3
 Bundle-SymbolicName: com.prolan.tycho.release3
-Bundle-Version: 1.0.0.qualifier
+Bundle-Version: 1.1.1
 Export-Package: com.prolan.tycho.release3
